requirements.txt

```diff
@@ -1,8 +1,9 @@
+attrs==23.1.0
```

Comment:
Yeah I think the old openai library must have been requiring it because I deleted my env and installed requirements.txt and it suddenly wasn't installed anymore haha